confmap/provider.go

```diff
@@ -105,7 +105,7 @@ func WithRetrievedClose(closeFunc CloseFunc) RetrievedOption {
 // * CloseFunc specifies a function to be invoked when the configuration for which it was
```

Comment:
the docstring for `cfgMap` needs updating

Reply:
Done.